### PR TITLE
Address

### PR DESCRIPTION
**Pull request by rofe**
Extracted `.section.contact-us` styling to a dedicated `address` block.

Test URL: https://address--tlcgrp--hlxsites.hlx.page/drafts/rofe/address

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -324,44 +324,6 @@ main .section .default-content-wrapper h1 {
...
-
-main .section.contact .default-content-wrapper p {
-  margin-bottom: 0;
-}
-
-main .section.contact .default-content-wrapper em {
-  font-size: 1em;
-  padding-bottom: 0;
-}
-
```
Comment (asthabh23) [inline styles/styles.css:335]: These changes are to style 'contact' section on /digitech page which are still needed.

File: styles/styles.css [hunk 2]
```diff
@@ -460,19 +422,6 @@ main .section.align .default-content-wrapper h4 {
...
-  }
-
-  main .section.contact .default-content-wrapper p {
-    margin-bottom: 0;
-  }
-
-  main .section.contact .default-content-wrapper em {
-    font-size: 1.2em;
-    padding-bottom: 0;
-  }
```
Comment (asthabh23) [inline styles/styles.css:465]: These changes are to style 'contact' section on /digitech page which are still needed.

**Review by asthabh23 — CHANGES_REQUESTED** [review]
We need to retain changes for 'contact' section (related to /digitech page)

**Comment by rofe** [comment]
> We need to retain changes for 'contact' section (related to /digitech page)

@asthabh23 I created a dedicated block for this style, see https://address--tlcgrp--hlxsites.hlx.page/digitech.

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**